### PR TITLE
Update to new Dokka 

### PR DESCRIPTION
**Pull request by qwwdfsad**

### CODE REVIEW

**Comment by qwwdfsad** [comment]
The new site is also online https://kotlin.github.io/kotlinx.serialization/

File: docs/json.md [hunk 1]
```diff
@@ -483,8 +483,8 @@ it explains, among other things, how custom serializers are bound to classes.
 Transformation capabilities are provided by the abstract [JsonTransformingSerializer] class which implements [KSerializer]. 
 Instead of direct interaction with `Encoder` or `Decoder`, this class asks you to supply transformations for JSON tree 
 represented by the [JsonElement] class using the 
-[transformSerialize][JsonTransformingSerializer.transformSerialize] and 
-[transformDeserialize][JsonTransformingSerializer.transformDeserialize] methods. Let us take a look at the examples.
+`transformSerialize` and
```
Comment (sandwwraith) [inline docs/json.md:486]: why can't we reference it here?

File: dokka/moduledoc.md [hunk 2]
```diff
@@ -1,31 +1,52 @@
-# Module kotlinx-serialization-runtime
-
-A companion library to kotlinx.serialization compiler plugin.
-Contains core serialization API, serializers for standard library classes, and ready to use JSON
+# Module kotlinx-serialization-core
+Core serialization API, serializers for standard library classes, and ready to use JSON
```
Comment (sandwwraith) [inline dokka/moduledoc.md:2]:
```suggestion
Core serialization API and serializers for standard library classes
```

File: dokka/moduledoc.md [hunk 3]
```diff
@@ -1,31 +1,52 @@
...
 format implementation.
-Other formats implementations (Protobuf, CBOR) are provided by separate artifacts, but included to this document for integrity.
 
-# Package kotlinx.serialization
+# Module kotlinx-serialization-json
+Stable and ready to use JSON format implementation, `JsonElement` API to operate with JSON trees and JSON-specific serializers.
+
+# Module kotlinx-serialization-cbor
+Concise Binary Object Representation (CBOR) format implementation, as per [RFC 7049](https://tools.ietf.org/html/rfc7049).
```
Comment (sandwwraith) [inline dokka/moduledoc.md:9]: Maybe add something about experimentality?

File: dokka/moduledoc.md [hunk 4]
```diff
@@ -1,31 +1,52 @@
...
-
 Serializers for standard Kotlin types, like Int, String, List, etc.
 
-# Package kotlinx.serialization.modules
+# Package kotlinx.serialization.descriptors
+Basic concepts of serial description to programmatically describe the serial form for serializers 
+in an introspectable manner.
 
+# Package kotlinx.serialization.encoding
+Basic concepts of data encoding and decoding of serialized data.
```
Comment (sandwwraith) [inline dokka/moduledoc.md:33]:
```suggestion
Basic concepts of encoding and decoding of serialized data.
```

Duplicated word 'data' does not sound quite right

File: formats/hocon/build.gradle [hunk 5]
```diff
@@ -17,6 +17,7 @@
 apply plugin: 'kotlin'
 apply plugin: 'kotlinx-serialization'
 
+
```
Comment (sandwwraith) [inline formats/hocon/build.gradle:20]: not sure it was necessary but ok

File: update_docs.sh [hunk 6]
```diff
@@ -38,16 +38,21 @@ cd "$PAGES_DIR"
...
-# Copy new documentation from dist
+# Copy manually new documentation and flat out kotlinx-serialization
 cp -r "$DIST_DIR"/* "$PAGES_DIR"
 
+# Hack for the GH pages
+cp $PAGES_DIR/-modules.html $PAGES_DIR/index.html
+
 # Add it all to git
-git add *
+#git add *
```
Comment (sandwwraith) [inline update_docs.sh:53]:
```suggestion
# git add *
```

**Review by sandwwraith — APPROVED** [review]

File: docs/json.md (hunk 1 above)
Comment (qwwdfsad) [reply docs/json.md:486]: It's a Dokka bug: https://github.com/Kotlin/dokka/issues/1522

**Pull request merged**